### DISCUSSION TITLE
Cannot get verified with unconditional offer

### DISCUSSION THREAD

**Discussion by Chivier** [community/community#55433] [GitHub Education]
### Hi everyone,

Hi Github,

I was in a 'transition period' between my graduate student life and PhD life. I just received an unconditional offer a few weeks ago. And I have been sending different files to you during these weeks. I tried the offer letter, the school website, the student application hub website snapshot. But I still fail to renew my student pack.

I can receive my Student card after I go to the University in this Octobor, but github student pack is essential to my daily life. I use it to work together with my friends everyday. I am wondering what else can I submit to you to prove I am a student.

### A suggestion...(optional)

(BTW, I think a pdf/png/jpeg submitssion button is necessary. I have to use obs virtual camera to take screenshot, and this is really uncovenient!)

_No response_

**Comment by samus-aran** [comment]
Hi @Chivier 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
